FirebaseSessions/Sources/Development/DevEventConsoleLogger.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+class DevEventConsoleLogger: EventGDTLoggerProtocol {
+  private let commandLineArgument = "-FIRSessionsDebugEvents"
+
+  func logEvent(event: SessionStartEvent, completion: @escaping (Result<Void, Error>) -> Void) {
+    if !ProcessInfo.processInfo.arguments.contains(commandLineArgument) {
+      return
+    }
+
+    let proto = event.encodeDecodeEvent()
+    prettyPrint(proto: proto)
+  }
+
+  func prettyPrint(proto: firebase_appquality_sessions_SessionEvent) {
+    let logOutput = """
+    Logging Session Event due to \"\(commandLineArgument)\" command line argument
```

Comment:
So I looked into this. NanoPB doesn't have an api that gives us the name of fields, and it doesn't seem to store field names either.

I tried something like:

```
    var protoVar = event.encodeDecodeEvent()
    var iter: pb_field_iter_t = pb_field_iter_t()
    var fields = firebase_appquality_sessions_SessionEvent_fields
    pb_field_iter_begin(&iter, &fields.0, &protoVar)
    repeat {
      let typeHex = String(format:"0x%02X", iter.pos.pointee.type)
      print("HI: \(iter.pos.pointee.type), \(typeHex)")
    } while(pb_field_iter_next(&iter));
```

This would iterate over all the fields. Unfortunately the field object doesn't have names. Just the tags.